modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -637,6 +637,9 @@ public Response getByLuceneQuery(
       query.withQuery(q);
     }
 
+    // Include series data in the results?
+    query.includeSeries(includeSeries);
```

Comment:
I've added this to aid debugging when not using a standalone solr setup. This way you can search for series as well using the the lucene endpoint.